app/dashboard/templates/dashboard/index.html

```diff
@@ -53,6 +53,11 @@
           {% endif %}
           <span class="d-block font-title-lg">
             {{ hackathon.start_date|date:"M j, Y" }} - {{ hackathon.end_date|date:"M j, Y" }}
+            {% if hackathon.end_date|timeuntil >= "1 min" %}
```

Comment:
Does the comparison here actually work? I don't think something like "8 hours" >= "1 min" works with string comparisons, but if im wrong I will be amazed at Django's usefulness :) If it turns out we need better comparison logic, according to the Django docs `timeuntil` will return the string "0 minutes" if the end date here is already passed so we can just check `{% if hackathon.end_date|timeuntil != "0 minutes" %}` https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#timeuntil

Reply:
thats crazy! magic django!